apps/web/src/components/tx/SignOrExecuteForm/SignOrExecuteFormV2.tsx

```diff
@@ -133,11 +142,9 @@ export const SignOrExecuteFormV2 = ({
   }
 }
 
-const useSafeTx = () => useContext(SafeTxContext).safeTx
-const useSafeTxError = () => useContext(SafeTxContext).safeTxError
+const useSafeTxContext = () => useContext(SafeTxContext)
 
 export default madProps(SignOrExecuteFormV2, {
   chainId: useChainId,
-  safeTx: useSafeTx,
-  safeTxError: useSafeTxError,
+  safeTxContext: useSafeTxContext,
```

Comment:
I don't think this change is necessary (and the prop destructuring) after 0d47849c5ec8528c4b099dad3c4f5291fa33de4b? It would reduce the diff.